aleph0/src/createActions.ts

```diff
@@ -0,0 +1,129 @@
+import { z } from "zod";
+import { loadSnippets } from "./lib/utils";
+import { createFile, createFolder, readFile } from "./lib/file";
+import { Technology } from "./types";
+import { RunnableFunctionWithParse } from "openai/lib/RunnableFunction.mjs";
+
+export const createActions = (options: {
+  technology: Technology;
+}): Record<string, RunnableFunctionWithParse<any>> => {
+  const { technology } = options;
```

Comment:
not sure why this is passed as an option. should be an `arg`, right?

Reply:
I thought we wanted to indicate to the ai about the technology we are using.